### PR TITLE
Hotfix qiskit

### PR DESCRIPTION
**Pull request by KilianPoirier**
## Description

- Fix for the qiskit breaking changes
- **Fixes # 296**

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py [hunk 1]
```diff
@@ -22,6 +24,18 @@ def setUp(self):
                 for each_subclass_gate in qsk_c_gate.ControlledGate.__subclasses__()
             ]
         )
+        # available_gates.extend(
```
Comment (KilianPoirier) [inline src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py:27]: Same issue here with the Singleton gates

File: src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py [hunk 2]
```diff
@@ -3,6 +3,8 @@
 from qiskit.circuit import gate as qsk_gate
 from qiskit.circuit import controlledgate as qsk_c_gate
 from qiskit.circuit.library import standard_gates as qsk_s_gate
+
+# from qiskit.circuit import singleton as qsk_sgt
```
Comment (KilianPoirier) [inline src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py:7]: The XGate was displaced to qiskit.circuit.Singleton. All non-parametrized gates have been changed to Singleton objects and passed only by reference to the overarching circuit.
It was later resolved (I'm assuming that it's now accessible through qiskit.circuit.gate) and therefore left it as a comment.

Comment (vishal-ph) [reply src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py:7]: so, the previous imports work again now?

Comment (vishal-ph) [reply src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py:7]: I have already approved the PR, we can merge it once we resolve this conversation

**Review by vishal-ph — APPROVED** [review]

File: src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py (hunk 2 above)
Comment (KilianPoirier) [reply src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py:7]: The error was that the XGate (unparametrized) had been moved from `qiskit.circuit.gate` to `qiskit.circuit.singleton`. Which apparently was later changed such that the tests would pass again. I ran the tests again to make sure that they still function correctly.

**Pull request merged**